modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java

```diff
@@ -894,25 +869,24 @@ private void _verifyMembershipPolicies() throws Exception {
 	private static final Log _log = LogFactoryUtil.getLog(
 		EditServerMVCActionCommand.class);
 
+	@Reference
+	private static ClusterExecutor _clusterExecutor;
+
+	@Reference
+	private static ClusterMasterExecutor _clusterMasterExecutor;
+
 	private static final Function<InvocationHandler, Layout>
 		_proxyProviderFunction = ProxyUtil.getProxyProviderFunction(
 			Layout.class, ModelWrapper.class);
 	private static final MethodKey _resetLogLevelsMethodKey = new MethodKey(
 		EditServerMVCActionCommand.class, "_resetLogLevels", Map.class,
 		Map.class);
 	private static final MethodKey _updateLogLevelsMethodKey = new MethodKey(
-		EditServerMVCActionCommand.class, "_updateLogLevels", Map.class,
-		String.class);
+		EditServerMVCActionCommand.class, "_updateLogLevels", Map.class);
```

Comment:
@RenanUmV Why did you change the method reference?

Reply:
Can you please point to the pattern you are following here?

Reply:
I changed the method reference because the method that was used before was deleted, in the method I'm using now it only receives a Map as a parameter. I'm following the pattern that Dante applied in this [PR](https://github.com/liferay-core-infra/liferay-portal/pull/4718/commits/4fded4a90a29f6e7b7aab446185156bab565b7b5)

Reply:
got it

Reply:
send it to core-infra repo ant explain that there as well